src/course-team/add-team-member/AddTeamMember.scss

```diff
@@ -0,0 +1,24 @@
+.add-team-member {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border: .0625rem solid $gray-200;
+  border-radius: .375rem;
+  box-shadow: inset 0 1px 2px 1px $gray-200;
+  padding: 20px 30px;
+
+  .add-team-member-info {
+    width: 60%;
+  }
+
+  .add-team-member-title {
+    font-size: 16px;
+    font-weight: 600;
+  }
+
+  .add-team-member-description {
+    font-size: 14px;
```

Comment:
let's use typography from [paragon](https://paragon-openedx.netlify.app/foundations/typography) lib
use the most appropriate one if you don't have a layout.

please check it everywhere in your pr

Reply:
Done